libs/common-traits/src/lib.rs

```diff
@@ -100,9 +100,13 @@ pub trait BigEndian<T> {
 
 /// A trait that can be used to fetch the nav and update nav for a given pool
 pub trait PoolNAV<PoolId, Amount> {
+	type ClassId;
+	type Origin;
 	// nav returns the nav and the last time it was calculated
 	fn nav(pool_id: PoolId) -> Option<(Amount, u64)>;
 	fn update_nav(pool_id: PoolId) -> Result<Amount, DispatchError>;
+	fn initialise(origin: Self::Origin, pool_id: PoolId, class_id: Self::ClassId)
```

Comment:
I really don't like this, but right now we're in the situation where the `loans` pallet already depends on `pools`, so in order to work with the `loans` pallet from the `pools` benchmarks, we need to go through a trait to break the dependency cycle.

We could make a `NAVBenchmarking` trait that has this method and is only implemented when benchmarks are enabled, but that feels even clunkier and more error-prone.